test-cloud/springboot/test-springboot/src/test/resources/kjars-sources/definition-project/pom.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie.server.testing</groupId>
+  <artifactId>definition-project</artifactId>
+  <version>1.0.0.Final</version>
+  <name>definition-project</name>
+</project>
```

Comment:
Shouldn't we add `kie-maven-plugin` plugin as part of the build? Not sure, wdyt?

Reply:
Well, ideally yes but it brings also negative sides.
The Maven repo created while building the kjar needs to be copied to the image too. In case of current setup the repo has 7.7 MB, in case the plugin is added then repo size raise to 92.6 MB. Also downloading all the artifacts extends test execution time (can be largely compensated by using mirror).
IMHO using plugin is not needed in this case as it doesn't bring us any significant benefit.